### ISSUE TITLE
cls-bluebird@1.0.1 requires a peer of continuation-local-storage@~3 but none was installed

### ISSUE THREAD

**Issue by satellite1** [blockchain/service-my-wallet-v3#78]
Ubuntu 14.04 with the following:
 *Service v0.19.0, Node v5.8.0*

Getting Unmet Peer Dependency continuation-local-storage@~3
cls-bluebird@1.0.1 requires a peer of continuation-local-storage@~3 but none was installed

Note:
Service [blockchain-wallet-service start --port 3000] does start without error.

Thanks!

**Comment by jtormey** [comment]
Hi, since this is a peer dependency, it's not directly used by the service. Because of that, there shouldn't be any issues when it isn't installed.

**Comment by jtormey** [comment]
What issues are you seeing in this version?

**Comment by satellite1** [comment]
At first I encountered the Address GAP 20 issue during testing. It would be great to make that notation clear in the documentation to save a lot of others a lot of time. When that was remedied, I attempted to post transactions [receive to a new wallet and new account with new xpub], and now I'm getting wallet.createPayment is not a function. [Attached screenshot]
![log_blockchain](https://cloud.githubusercontent.com/assets/16124737/13862200/2261d97a-ec68-11e5-88d9-8ce15d768c18.jpg)

Thanks for the prompt reply and assistance!

**Comment by satellite1** [comment]
Appears CLS no required dependency for blockchain service, errors related to Address GAP. 
Thanks for clarification!

**Comment by jtormey** [comment]
After fixing the address gap error, do you stop seeing the wallet.createPayment errors?

**Comment by satellite1** [comment]
A few things: Created a new account [guid] and xpub, cleared my local db of all test transactions, rebooted the sever, and restarted the service. [making sure nothing in memory or residual from any place else.] Then on subsequent tests, "yes" error was removed; however, I then ran into #31 that was rectified with the "from" parameter. I'll update when a final end-to-end test with v.19 receive and send completes without error. Thanks for the follow-up.
